presto-hive-hadoop2/bin/common.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
 function retry() {
```

Comment:
we should have a single instance of this. 

Maybe we should have some commons library for bash somewhere.